homeassistant/components/alarm_control_panel/group.py

```diff
@@ -0,0 +1,236 @@
+"""
+Group platform for alarm control panel component.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/alarm_control_panel.group/
+"""
+import asyncio
+from collections import Counter
+from copy import deepcopy
+import logging
+
+import voluptuous as vol
+
+from homeassistant.core import callback
+import homeassistant.components.alarm_control_panel as alarm
+from homeassistant.components.alarm_control_panel import (
+    DOMAIN, PLATFORM_SCHEMA, SERVICE_ALARM_DISARM, SERVICE_ALARM_ARM_HOME,
+    SERVICE_ALARM_ARM_AWAY, SERVICE_ALARM_ARM_NIGHT, SERVICE_ALARM_TRIGGER,
+    SERVICE_ALARM_ARM_CUSTOM_BYPASS)
+from homeassistant.const import (
+    ATTR_ENTITY_ID, ATTR_PRE_PENDING_STATE, ATTR_POST_PENDING_STATE,
+    CONF_ENTITIES, CONF_NAME, EVENT_HOMEASSISTANT_START,
+    STATE_ALARM_ARMED_CUSTOM_BYPASS, STATE_ALARM_PENDING,
+    STATE_ALARM_TRIGGERED)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.event import async_track_state_change
+
+ATTR_TRIGGERED_PANELS = 'triggered_panels'
+
+CONF_CODE_FORMAT = 'code_format'
+
+DEFAULT_ALARM_NAME = 'Group Alarm'
```

Comment:
Please also change the default name to `Alarm Group` for consistency among group platforms.